redis/query_processor.hh

```diff
@@ -25,6 +25,7 @@
 #include <seastar/core/shared_ptr.hh>
 #include <seastar/core/gate.hh>
 #include <seastar/core/metrics_registration.hh>
+#include "seastarx.hh"
```

Comment:
I think @penberg (?) deliberately added `seastar::` qualifiers throughout this header file, **instead** of adding "seastarx.hh". It's strange that you added below "smp_service_group" without "seastar::", when the line above it and below it have explicit "seastar::".
Let's try to be consistent at least inside one source file, if we can.

Reply:
@nyh Yes I indeed did. The preference for Redis code is to use fully qualified names, so include `seastar::` please, @syuu1228.

Reply:
Dropped `#include "seastarx.hh"`, use `seastar::smp_service_group`.